app-shells/rc/rc-1.7.4-r2.ebuild

```diff
@@ -28,16 +28,15 @@ src_configure() {
 }
 
 src_install() {
-	into /usr
-	newbin "${PN}" "${PN}sh"
-	newman "${PN}.1" "${PN}sh.1"
+	into  "${EROOT}/usr"
```

Comment:
EROOT is wrong here and the original was correct. Indeed, pkgcheck should warn about this too.

'into' is already using EPREFIX anyway.

Reply:
> Indeed, pkgcheck should warn about this too.

Hm, I've not seen that 🤔. By the way, thanks for the explanation, and sorry for the misunderstanding.